docs/index.md

```diff
@@ -6397,6 +6398,19 @@ The remaining transaction types do not have any flags at this time.
 * PaymentChannelCreate
 * PaymentChannelFund
 
+## txFlagIndices
+
+The following are integer (as opposed to bit) flags that can be set for particular transactions in the AccountSet SetFlag or ClearFlag field
+
+`txFlagIndices.AccountSet.asfRequireDest`: Require a destination tag to send transactions to this account.
```

Comment:
Perhaps something like this (will require renames/aliases elsewhere):
```suggestion
`RippleAPI.asfRequireDest`: Require a destination tag to send transactions to this account.
```

In xrpl.js, this could be `XRPL.asfRequireDest`.

Reply:
"asf" is already short for "AccountSet flag", I believe.